src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,55 @@
+package lotto.domain;
+
+import java.util.*;
+
+public class Lotto {
+
+    public static final int LOTTO_COUNT = 6;
+    public static final int MIN_NUM = 1;
+    public static final int MAX_NUM = 45;
+    public static final int PRICE = 1000;
+
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        validLottoNumberRange(numbers);
+        validLottoNumberCount(numbers);
+        validDuplicate(numbers);
+        this.numbers = new ArrayList<>(numbers);
+    }
+
+    private void validLottoNumberRange(List<Integer> numbers) {
+        Integer max = Collections.max(numbers);
+        Integer min = Collections.min(numbers);
+        if(max > MAX_NUM || min < MIN_NUM) {
```

Comment:
IDE로 IntelliJ IDEA를 사용하신다면 자동 포맷팅 단축키인 `Ctrl + Alt + L` 활용을 추천드립니다.
```suggestion
        if(max > MAX_NUM || min < MIN_NUM) {
```